pkgs/applications/window-managers/picom/picom-allusive.nix

```diff
@@ -0,0 +1,24 @@
+{ picom, lib, fetchFromGitHub }:
+
+picom.overrideAttrs (oldAttrs: rec {
+  pname = "picom-allusive";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "allusive-dev";
+    repo = "picom-allusive";
+    rev = version;
+    hash = "sha256-lk4Ll0mi9h3BAqwgOzFQw4WYKnSW9XTl3PjoK2E4WKg=";
+  };
+
+  postInstall = ''
+    chmod +x $out/bin/picom-trans
+  '' + oldAttrs.postInstall;
+
+  meta = with lib; {
```

Comment:
minor nitpick, but nixpkgs is trying to drop the redundant usage of `with lib;` and this could be replaced with lib.licenses and lib.maintainers

Reply:
I tried using lib.licences before but that resulted in errors.

Reply:
Ok, i made the changes and pushed again.